src/backend/distributed/master/master_create_shards.c

```diff
@@ -165,7 +165,7 @@ CreateShardsWithRoundRobinPolicy(Oid distributedTableId, int32 shardCount,
 	HOLD_INTERRUPTS();
 
 	/* retrieve the DDL commands for the table */
-	ddlCommandList = GetTableDDLEvents(distributedTableId);
+	ddlCommandList = GetTableDDLEvents(distributedTableId, true);
```

Comment:
We have a convention to use variable names instead of constants in places like this. Not everyone's happy about this convention, but it does make the meaning of parameters a little easier to read on GitHub.